piet-cairo/README.md

```diff
@@ -0,0 +1,29 @@
+# piet-cairo: Cairo backend for piet
+
+This is the [Cairo](https://www.cairographics.org/) back-end for the piet graphics API.
+
+## Toy text API
+
+For simplicity, the back-end currently uses the [toy text API] in Cairo. Essentially, this means there is no shaping, so complex scripts won't render correctly at all, and Latin will be missing kerning, ligatures, and other refinements. According to the docs, "Any serious application should avoid them."
+
+Fairly soon, I hope to have some type of higher-level text in place. One possibility is [pango]. From what I can tell, this should work well on Linux, but since it has a non-optional glib dependency, it might be non-trivial to get it building portably. It's also not clear to me how well this approach handles discovering system fonts.
+
+Another possibility is to use HarfBuzz more directly, using the [rust-harfbuzz] bindings. This will require more work for font discovery, but has the possibility to be considerably more native.
+
+A third possibility is to adapt [libTXT] from Flutter. This is a state of the art text layout library, with considerable investment in making it work well on mobile. However, it is in C++ and thus at the very least will need nontrivial work to make good Rust bindings.
```

Comment:
Should an issue be opened for discussing which text layout engine to use? I have questions about this, but this PR doesn't seem like the right place for this discussion.

Reply:
Yes, another issue for that is a great idea. Let's discuss it there.